docs/zh-CN/administrator-guide/config/be_config.md

```diff
@@ -1160,8 +1160,8 @@ storage_flood_stage_usage_percent和storage_flood_stage_left_capacity_bytes两
 
   `storage_root_path=/home/disk1/doris.HDD,50;/home/disk2/doris.SSD,10;/home/disk2/doris`
 
-  * /home/disk1/doris.HDD, 50，表示存储限制为50GB, HDD;
-  * /home/disk2/doris.SSD 10， 存储限制为10GB，SSD；
+  * /home/disk1/doris.HDD,50，表示存储限制为50GB，HDD;
+  * /home/disk2/doris.SSD,10，存储限制为10GB，SSD；
```

Comment:
It's ok to use Chinese symbol in Chinese.
It's not essential to modify it.

Reply:
It doesn't concern the Chinese symbol, please refer the issue #7619.